monks-tokenbar.js

```diff
@@ -376,7 +376,7 @@ export class MonksTokenBar {
     }
 
     static getDiceSound(hasMaestroSound = false) {
-        const has3DDiceSound = game.dice3d ? game.settings.get("dice-so-nice", "settings").enabled : false;

```

Comment:
This is an old way to check for dice-so-nice and is basically no longer supported.

```javascript
    //Not used anymore but kept for compatibility with migration
    game.settings.register("dice-so-nice", "settings", {
        name: "3D Dice Settings",
        scope: "client",
        default: {},
        type: Object,
        config: false
    });
```